cmd/updatehub/main.go

```diff
@@ -132,3 +140,23 @@ func main() {
 
 	os.Exit(d.Run())
 }
+
+func sanitizeServerAddress(address string, disableHTTPS bool) (string, error) {
+	a := address
+	if !strings.HasPrefix(a, "http://") && !strings.HasPrefix(a, "https://") {
+		a = "https://" + a
```

Comment:
The disableHTTPS option always overwrite the "scheme" passed on the URL, so it will work on both cases.